src/mongoose_acc.erl

```diff
@@ -232,10 +302,11 @@ default_non_strippable() ->
      origin_pid,
      origin_location,
      origin_stanza,
+     stanza,
+     lserver,
```

Comment:
What exactly does `lserver` represent here? The `lserver` value of:
1. Sender JID,
2. Recipient JID,
3. JID associated with C2S processing the ACC?

Reply:
lserver of the initiator (creator of the Acc). for the Acc created by C2S on some incoming stanza that would be sender jid value. it is mandatory field and I believe it's added for the tracing/debugging purposes, e.g. if some module create new Acc w/o stanza it's still possible to figure out lserver of the module.